ocs_ci/ocs/ui/block_pool.py

```diff
@@ -89,16 +98,20 @@ def delete_pool(self, pool_name):
             bool: True if pool is not found in pool list, otherwise false
 
         """
-
         self.navigate_overview_page()
         self.navigate_block_pool_page()
         self.page_has_loaded()
-        self.do_click((f"{pool_name}", By.LINK_TEXT))
+        if self.ocp_version_full <= version.VERSION_4_8:
+            self.do_click((f"{pool_name}", By.LINK_TEXT))
+        else:
+            self.do_click((f"//a[text()='{pool_name}']", By.XPATH))
```

Comment:
Can you work with the element dictionary.?
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/views.py
I prefer to add "HTML" code only on views.py file

Reply:
@OdedViner in the above suggestion, the value takes the pool name as a variable value. I think it will be difficult to pass the value of pool name to the element dictionary, as the pool name will vary everytime

Reply:
I have added a function in helpers_ui

Reply:
but you don't use it here